rest-api-spec/src/main/resources/rest-api-spec/test/indices.data_stream/10_basic.yml

```diff
@@ -62,62 +80,56 @@
 ---
 "Create data stream with invalid name":
   - skip:
-      version: " - 7.8.99"
-      reason: "data streams only supported in 7.9+"
+      version: " - 7.99.99"
+      reason:  "mute bwc until backported"
 
   - do:
       catch: bad_request
       indices.create_data_stream:
         name: invalid-data-stream#-name
```

Comment:
The check whether a template with the same exists happens before validating the name of the template. I can check the error description, so that we ensure that this request fails because of an invalid name.